### PR TITLE
feat: added support to run cli with docker run cmd + added try catch i…

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
…n validations

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,21 +29,8 @@ RUN --mount=type=bind,source=docker/patch/redis.py,target=redis.py \
...
-RUN npm run build
-RUN cp -r dist/* ./
-
-# Creating user to simulate openshift.
-RUN useradd -ms /bin/bash user && usermod -a -G root user
-USER user
-
-# Expose the port that the application listens on.
-EXPOSE 8080
+# COPY --from=build /tmp/buildApp/* ./
```
Comment (asafMasa) [inline Dockerfile]: remove

File: src/seed.ts [hunk 2]
```diff
@@ -16,14 +15,14 @@ export type SeedOptions = {
...
 export const executeSeed = async (options: SeedOptions): Promise<void> => {
-  const mapproxyYamlFilePath = config.get<string>('mapproxyYamlFilePath');
-  const seedYamlFilePath = config.get<string>('seedYamlFilePath');
+  const mapproxyYamlFilePath = process.env.MAPPROXY_YAML_FILE_PATH ?? '/mapproxy/mapproxy.yaml';
+  const seedYamlFilePath = process.env.SEED_YAML_FILE_PATH ?? '/mapproxy/seed.yaml';
   const flags: string[] = [];
 
   console.log('validating supported cache.');
   validateSupportedCache(mapproxyYamlFilePath, options.cache);
   console.log(`creating seed yaml file on path: ${seedYamlFilePath}.`);
```
Comment (asafMasa) [inline src/seed.ts:24]: add log for mapproxy to

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply Dockerfile]: removed

File: src/seed.ts (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/seed.ts:24]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**